server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java

```diff
@@ -156,6 +156,64 @@ void fullReindexingShouldCompleteWhenNoMail() {
             .body("additionalInformation.messageFailures", is(anEmptyMap()));
     }
 
+    @Test
+    void recomputeFastViewProjectionItemsShouldCompleteWhenNoMail(GuiceJamesServer server) throws Exception {
+        server.getProbe(DataProbeImpl.class).addUser(USERNAME, "secret");
+        mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, USERNAME, MailboxConstants.INBOX);
+        mailboxProbe.appendMessage(
+                USERNAME,
+                MailboxPath.inbox(Username.of(USERNAME)),
+                new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()),
+                new Date(),
+                false,
+                new Flags());
+
+        String taskId = with()
+            .post("/mailboxes?task=recomputeFastViewProjectionItems")
+            .jsonPath()
+            .get("taskId");
```

Comment:
indent?

Reply:
Nope we indent given/when/then not with calls.